--- src/Makefile	2009-01-11 13:17:37.000000000 +0100
+++ src/Makefile	2010-09-01 12:36:28.975639416 +0200
@@ -10,7 +10,7 @@
 
 .c.o:
 	echo $*.c:
-	gcc -c -O2 -pipe \
+	i586-mingw32msvc-gcc -c -O2  \
 	-falign-functions -fomit-frame-pointer -fno-strict-aliasing \
 	-W -Wimplicit -Wreturn-type -Wunused -Wformat \
 	-Wuninitialized -Wstrict-prototypes \
@@ -21,8 +21,8 @@
 $(bin)/picolisp: $(picoFiles:.c=.o)
 	mkdir -p $(bin)
 	echo "  " link picolisp:
-	gcc -o $(bin)/picolisp $(picoFiles:.c=.o) -lc -lm
-	strip $(bin)/picolisp
+	i586-mingw32msvc-gcc -o $(bin)/picolisp.exe $(picoFiles:.c=.o)  -lm
+	i586-mingw32msvc-strip $(bin)/picolisp.exe
 
 
 # Clean up
